tests/foreman/performance/common/perf_concurrent_test.py

```diff
@@ -0,0 +1,372 @@
+"""Test utilities for writing performance tests
+
+Define ConcurrentTestCase as parent class of writting Subscribe by AK,
+Subscribe by register and attach, Subscription Deletion.
+
+"""
+
+from robottelo.common import ssh
+
+import unittest
+import logging
+import csv
+
+from tests.foreman.performance.common.perf_thread \
```

Comment:
We need to avoid `\` to break lines, by moving the module will make this import fit in one line.